stix_shifter_modules/nozomi/stix_transmission/connector.py

```diff
@@ -0,0 +1,217 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_json_sync_connector import BaseJsonSyncConnector
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from stix_shifter_utils.utils import logger
+from .api_client import APIClient
+import json
+import re
```

Comment:
regex is faster than the built in re function.

Reply:
Updated as per the suggestion.